### PR TITLE
Delegate signature

### PR DESCRIPTION
**Pull request by alicevhenshaw**
## Description

Updating the Delegate to require a signature on orders. Per issue https://github.com/airswap/airswap-protocols/issues/323

Quick description:

- [X] Typo/small tweaks

## Changes

- Updated the require statement in the delegate
- Added signatures to tests that previously didnt provide them
- Added 2 new tests

## Tests

Delegate: `  93 passing (17s)`

## Test Coverage

Unchanged

### CODE REVIEW

File: source/delegate/contracts/Delegate.sol [hunk 1]
```diff
@@ -219,8 +219,8 @@ contract Delegate is IDelegate, Ownable {
 
     Rule memory rule = rules[order.sender.token][order.signer.token];
 
-    require(order.signer.wallet == msg.sender,
-      "SIGNER_MUST_BE_SENDER");
+    require(order.signature.v != 0,
+      "SIGNATURE_MUST_BE_SENT");
```
Comment (poiresel) [inline source/delegate/contracts/Delegate.sol]: SIGNATURE_MUST_BE_SET?

File: source/delegate/test/Delegate.js [hunk 2]
```diff
@@ -910,8 +949,8 @@ contract('Delegate Integration Tests', async accounts => {
 
       // Succeeds on the Delegate, fails on the Swap.
```
Comment (poiresel) [inline source/delegate/test/Delegate.js]: Message doesn't match case "Fails on Delegate due to not sending signature"

File: source/delegate/contracts/Delegate.sol (hunk 1 above)
Comment (alicevhenshaw) [reply source/delegate/contracts/Delegate.sol]: We decided to keep this consistent with the wrapper and leave it as `SENT`.

File: source/delegate/test/Delegate.js [hunk 3]
```diff
@@ -1069,6 +1148,41 @@ contract('Delegate Integration Tests', async accounts => {
       )
     })
 
+    it('Gets a quote to sell 1 WETH where sender != signer, passes', async () => {
```
Comment (ejwessel) [inline source/delegate/test/Delegate.js]: did you want to add a test before this that tests the failure of having no signature?

Comment (alicevhenshaw) [reply source/delegate/test/Delegate.js]: Its on line 953

**Pull request merged**